src/components/loader/Loader.js

```diff
@@ -0,0 +1,5 @@
+import './loader.scss';
+
+export const Loader = ({ show }) => show && (
+  <div className="loader">&nbsp;</div>
```

Comment:
why need space here?

Reply:
I thought it is not correct to leave empty div tag )

Reply:
no, just use `<div />`